frontend/src/app/spot/styles/sass/components/list.sass

```diff
@@ -13,22 +13,42 @@
     &-action
       @include spot-body-big
 
+      position: relative
       display: flex
       align-items: center
       justify-content: flex-start
-      padding: $spot-spacing-0_5
+      padding: $spot-spacing-0_5 $spot-spacing-1
       background-color: transparent
       margin: 0
       border: 0
       cursor: pointer
 
       &:hover
-        background-color: $spot-color-main-light
+        background-color: $spot-color-basic-gray-6
+
+        .spot-list--item-hover
+          opacity: 1
 
       &_disabled
         color: $spot-color-basic-gray-3
         cursor: default
 
+    &-hover
```

Comment:
I would call this `hover-actions` or `actions`. Maybe @marcalcobe or @psatyal have an opinion, too.

Reply:
I agree. `Hover` is an activity not an object that can be selected or styled

Reply:
currently it results into an html like this:

```
<div class="spot-list--item-title" ...
<div class="spot-list--item-hover" ...
```

meaning, that the hover is defined right next to the content. As the parent is already called `spot-list--item-action` only `actions` seems weird. but I could fully go with `hover-actions`. Will talk to product team about the domain speech and how they would call this very piece of UI.

Reply:
After speaking with parimal, we agreed to adapt the name from the figma files, which is `-floating-actions`. This kind of decouples it from the trigger, which is hover, and emphasizes the actual behaviour of the component: floating over the sibling content.